### PR TITLE
Add featured posts to home page

### PR DESCRIPTION
**Pull request by gregjd**
This is the thing I worked on at the most recent session (at Learning Objects).

It has a bunch of old commits — most/all of which were previously merged with codefordc's version — but they're showing up here because of branching issues (I was having trouble pulling codefordc's commits into my branch). Rather than delaying while I fix the branch issues, I figured I'd just get this in here. But yeah, it's the most recent commit that's where the magic is.

### CODE REVIEW

File: ckanext/open_data_dc/templates/home/snippets/test.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<section class="module-content categories-list">
```
Comment (mkalish) [inline ckanext/open_data_dc/templates/home/snippets/test.html:1]: I think file name should change to more descriptive.

File: ckanext/open_data_dc/templates/snippets/least_popular_groups.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{# Renders a list of the site's least popular groups. #}
```
Comment (mkalish) [inline ckanext/open_data_dc/templates/snippets/least_popular_groups.html:1]: I don't think this is used?

**Review by mkalish — CHANGES_REQUESTED** [review]
I can show you how to rebase (remove the old commits) your changes back onto the master branch

**Comment by gregjd** [comment]
Replaced by https://github.com/codefordc/opendatadc/pull/61

**Pull request closed without merging**